great_expectations_cloud/agent/agent.py

```diff
@@ -156,7 +156,9 @@ def __init__(self: Self):
         with warnings.catch_warnings():
             # suppress warnings about GX version
             warnings.filterwarnings("ignore", message="You are using great_expectations version")
-            self._context: CloudDataContext = get_context(cloud_mode=True)
+            self._context: CloudDataContext = get_context(
+                cloud_mode=True, user_agent_str="gx-agent"
```

Comment:
Classic bikeshedding on a one-line PR. The intended format (copied from the PR that added the value) is `user_agent_str: Optional string, should be of format <PRODUCT> / <VERSION> <COMMENT>` where comment is optional. Can we do `user_agent_str=f"gx-agent / {__version__}}"`?

Reply:
sure